testkit/src/main/java/org/apache/calcite/test/catalog/MockCatalogReaderSimple.java

```diff
@@ -136,8 +116,9 @@ protected MockCatalogReaderSimple(RelDataTypeFactory typeFactory,
     empDefaultsTable.addColumn("DEPTNO", fixture.intTypeNull);
     empDefaultsTable.addColumn("SLACKER", fixture.booleanTypeNull);
     registerTable(empDefaultsTable);
+  }
 
-    // Register "EMP_B" table. As "EMP", birth with a "BIRTHDATE" column.
+  private void registerTableEmpB(MockSchema salesSchema, Fixture fixture) {
```

Comment:
Maybe renaming it to `registerTableEmpWithBirthday`? We have more space than in a table alias, it's better to be more explicit, especially because we are losing the comment after the refactoring.

Reply:
I move the comment to where we call this function. And I prefer to keep the name in style registerTable{tableName}, WDYT?

Reply:
Sounds reasonable, thanks for the explanation!